src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -1625,6 +1625,28 @@ public async Task GetAsync_UnicodeHostName_SuccessStatusCodeInResponse()
             }
         }
 
+        [Fact]
+        public async Task GetAsync_CustomTransferEncoding()
+        {
+            await LoopbackServer.CreateClientAndServerAsync(
+                async url =>
+                {
+                    using (HttpClientHandler handler = CreateHttpClientHandler())
+                    using (var client = new HttpClient(handler))
```

Comment:
Since you're not mutating the handler, you can just let HttpClient create it, e.g.
```C#
using (HttpClient client = CreateHttpClient())
```